docs/content/getting-started/production-environment.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Running OpenFGA in Production
+description: Running OpenFGA in a Production Environment
+sidebar_position: 8
+slug: /getting-started/running-in-production
+---
+
+import {
+  DocumentationNotice,
+  RelatedSection,
+} from "@components/Docs";
+
+# Running OpenFGA in Production
+
+<DocumentationNotice />
+
+## Preparation
+
+If you have tried OpenFGA locally, decided it works as expected, and now want to deploy it to production, we recommend the following preparation:
+
+- Update the configuration as follows:
+  - [Configure authentication](./setup-openfga.mdx#configuring-authentication) on the server.
+  - Enable TLS on the server.
+  - Enable production logging by changing the `log.format` in the [config.yaml](#configuring-the-server) configuration file to `json`.
+  - [Disable Playground](./setup-openfga.mdx#playground).
+- Add a rate limiter interceptor (for example, [in the server configuration](https://pkg.go.dev/github.com/openfga/openfga/server#Config)) that protects your OpenFGA server against bursts in traffic.
```

Comment:
This isn't really possible when running OpenFGA as a standalone server. We don't yet have a pluggable middleware mechanism that would allow developers to run OpenFGA in standalone mode with custom middleware plugins (such as rate limiting middleware).

Reply:
In this case, should we remove this (and the next) line?  What is our recommendation to the user?

Reply:
They can have a loadbalancer in front of it. The fact that it is standalone doesn't mean it has to be exposed to the public

Reply:
The recommendation points at config for the server that is non-configurable if running OpenFGA as a standalone server (not as an integrated server in some other binary).

If we're going to mention this then at minimum I recommend we change the verbiage to say something more like

> Enforce rate limits with upstream infrastructure (e.g. at your load-balancer or proxy layer)

Though this level of detail in an OpenFGA Production Checklist seems outside of the scope of this documentation. IMO our Production Checklist recommendations should include configurations specific to OpenFGA, not infrastructure elsewhere.

Reply:
In this case, let's rip these two lines out so that the document focuses on OpenFGA in a production system (not recommendation on how to setup production infrastructure in general).

Reply:
> our Production Checklist recommendations should include configurations specific to OpenFGA, not infrastructure elsewhere.

It is our responsibility to help people understand that they should not expose OpenFGA to the public without taking further action to secure their ecosystem.

Reply:
> In this case, let's rip these two lines out so that the document focuses on OpenFGA in a production system (not recommendation on how to setup production infrastructure in general).
👍 I agree. It's not our responsibility to try and give infrastructural guidelines for infrastructure we don't own. It is our responsibility to tell them how to configure OpenFGA to be secure and performant. If they want to put rate limiting in front of OpenFGA then that's up to them.

Reply:
Done